extensions/wikia/Wall/templates/WallHistory_index.php

```diff
@@ -125,7 +125,7 @@
 			<?= $app->renderView( 'PaginationController', 'index', [ 'totalItems' => $totalItems, 'itemsPerPage' => $itemsPerPage, 'currentPage' => $currentPage, 'url' => $wallHistoryUrl ] ); ?>
 		<?php endif;?>
 	<?php endif; ?>
-<?php else: ?>
+<? else: ?>
```

Comment:
So this change (and the one below) is not needed, right?

Reply:
grr... missed it :)